.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,8 @@
 <!-- What's in this pull request? -->
 Replace this paragraph with a description of your changes and rationale. Provide links to external references/discussions if appropriate.
 
-<!-- If this pull request resolves any bugs in the Swift bug tracker, provide a link: -->
-Resolves SR-NNNN.
+<!-- If this pull request resolves any issues in the Swift github, mention it: -->
```

Comment:
Maybe "resolves any issues in this repository"?

Reply:
Fair enough!

Reply:
> It doesn't have to be in this repo, as your link below shows, though it usually is

Speaking of, I get the feeling non-native speakers are liable to take "Swift github" for this repo. A plain "GitHub issues" is probably a better option. Thoughts?

Reply:
They probably will take it for this repo often, but that's why I gave an example of an externally linked repo too. "issues in the Github issues" is much worse: I think with the example below they will understand I mean the Swift umbrella, not just this repo.

Reply:
> "issues in the Github issues" is much worse

Sorry, I meant "If this pull request resolves any GitHub issues, mention it". I guess linking an issue originating in a third-party Swift repo is also acceptable, but—at your discretion.

Reply:
That's better, updated.

Reply:
Looks great, thank you!